### PR TITLE
Migrated ShiftingBoard.tsx from MaterialUI to Custom Components

### LINKED ISSUE #5000: Replace Material UI components in `src/Components/Shifting/ShiftingBoard.tsx`:
```tsx
8: import Button from "@material-ui/core/Button";\n11: import { Modal } from "@material-ui/core";\n13: import CircularProgress from "@material-ui/core/CircularProgress";
```

### Tasks

Ensure all the below tasks are accomplished if applicable to the file being modified in the PR.

- [ ] Migrate all Material UI components to use new Care UI / custom components. (ref: [🛠️ Tailwinding Care](https://github.com/coronasafe/care_fe/issues/3742))
- [ ] Migrate all Material UI icons to use [🏥 **Care Icons**](https://github.com/coronasafe/care_fe/pull/4071).
- [ ] Migrate `PageTitle` to `Page` component if any. (See [example](https://github.com/coronasafe/care_fe/pull/4796/files#diff-8e39f154ccf13c9dedd0f968d21a0abbde450e88df9724a4e7d4264d487a7846))

For more information see the epic issue that tracks this issue.

### PR DESCRIPTION
**Pull request by suprabathk**
## Proposed Changes
 
- [x]  Migrate all Material UI components to use new Care UI / custom components.
- [x]  Migrate all Material UI icons to use.
- [x]  Migrate PageTitle to Page component if any. 

- Fixes #5000 
- Change 1
  Migrated from all MaterialUI components to Custom made components.

- Change 2
  Changed the shade of text color in ConfirmDialogV2 component from 500 to 600 for better visibility.

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Shifting/ShiftingBoard.tsx [hunk 1]
```diff
@@ -204,60 +203,29 @@ const ShiftCard = ({ shift, filter }: any) => {
           </div>
           {filter === "TRANSFER IN PROGRESS" && shift.assigned_facility && (
             <div className="mt-2">
-              <Button
+              <ButtonV2
                 size="small"
```
Comment (rithviknishad) [inline src/Components/Shifting/ShiftingBoard.tsx]:
```suggestion
```

File: src/Components/Shifting/ShiftingBoard.tsx [hunk 2]
```diff
@@ -204,60 +203,29 @@ const ShiftCard = ({ shift, filter }: any) => {
...
-                variant="outlined"
-                fullWidth
+                variant="secondary"
+                border={true}
```
Comment (rithviknishad) [inline src/Components/Shifting/ShiftingBoard.tsx]: The button's width looks smaller than the View Details button which is right above it.

```suggestion
                border
                className="w-full"
```

**Review by rithviknishad — APPROVED** [review]
Just a few minor suggestions. The transfer button's width and size are not consistent with the view details button above it.

image

**Comment by bodhish** [comment]
copilot:summary

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**